### ISSUE TITLE
[Bug]: Connect failed when using pymilvus 2.2.8.dev3 and 2.2.8.dev2

### ISSUE THREAD

**Issue by NicoYuan1986** [milvus-io/milvus#23627]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version:2.2.0-20230423-b54afdbb
- Deployment mode(standalone or cluster):standalone
- MQ type(rocksmq, pulsar or kafka):    rocksmq
- SDK version(e.g. pymilvus v2.0.0rc2):2.2.8.dev3 2.2.8.dev2
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```

### Current Behavior

Connect failed when using pymilvus 2.2.8.dev3 and 2.2.8.dev2
```
―――――――――――――――――――――――――――――――――――― ERROR collecting testcases/test_collection.py ―――――――――――――――――――――――――――――――――――――
test_collection.py:5: in <module>
    from base.client_base import TestcaseBase
../base/client_base.py:7: in <module>
    from base.connections_wrapper import ApiConnectionsWrapper
../base/connections_wrapper.py:10: in <module>
    class ApiConnectionsWrapper:
../base/connections_wrapper.py:32: in ApiConnectionsWrapper
    def connect(self, alias=DefaultConfig.DEFAULT_USING, check_task=None, check_items=None, **kwargs):
E   AttributeError: type object 'Config' has no attribute 'DEFAULT_USING'
--------------------------------------------------- Captured stdout ----------------------------------------------------
[get_env_variable] failed to get environment variables : 'CI_LOG_PATH', use default path : /tmp/ci_logs
selected 0 items
```

### Expected Behavior

pass

**Comment by XuanYang-cn** [comment]
will be fixed by milvus-io/pymilvus#1387 and milvus-io/pymilvus#1388

**Comment by XuanYang-cn** [comment]
> will be fixed by [milvus-io/pymilvus#1387](https://github.com/milvus-io/pymilvus/pull/1387) and [milvus-io/pymilvus#1388](https://github.com/milvus-io/pymilvus/pull/1388)

Not ready, wait for milvus-io/pymilvus#1387 & milvus-io/pymilvus#1391

**Comment by XuanYang-cn** [comment]
should be fixed at latest 2.2.8.dev7
/unassign
/assign @NicoYuan1986

**Comment by NicoYuan1986** [comment]
fixed.
